projects/plugins/beta/src/class-hooks.php

```diff
@@ -300,7 +300,7 @@ private static function remove_dev_plugins( array $plugins ) {
 
 		// Initialize the WP_Filesystem API.
 		require_once ABSPATH . 'wp-admin/includes/file.php';
-		$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, false, array() );
+		$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, '', array() );
```

Comment:
Per [the docs](https://developer.wordpress.org/reference/functions/request_filesystem_credentials/), I think it should be this:
```suggestion
		$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, '', null );
```

That said, we're just using the defaults, so we could just use this:
```suggestion
		$creds = request_filesystem_credentials( site_url() . '/wp-admin/' );
```

Reply:
Passing null for `$extra_fields` isn't documented in the phpdocs, so Phan will complain about it, and also [actually means `array( 'version', 'locale' )` rather than `array()`](https://github.com/WordPress/wordpress-develop/blob/6fc019aa711308d6eaa1e672498b2a789a0c584c/src/wp-admin/includes/file.php#L2410-L2412).

It's not clear whether f902c8362877004fea5253752f0c4cdfec9dc634 used `array()` intentionally for the different default or what, beyond that they did specify it.